### PR TITLE
EDU-942 Create StepSlider component and use it where applicable

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-942

### CODE REVIEW

File: src/components/media-player/media-volume-slider.js [hunk 1]
```diff
@@ -37,10 +38,11 @@ function MediaVolumeSlider({ value, onChange, orientation }) {
       <Slider
```
Comment (adrianaluchian) [inline src/components/media-player/media-volume-slider.js:38]: Did you intentionally keep the direct `Slider` component usage here?

**Comment by adrianaluchian** [comment]
Nice consolidation! 👏 👍

File: src/components/media-player/media-volume-slider.js (hunk 1 above)
Comment (adrianaluchian) [reply src/components/media-player/media-volume-slider.js:38]: Got it

**Review by adrianaluchian — APPROVED** [review]

**Comment by ahelmberger** [comment]
> Nice consolidation! clap +1

Thanks!

**Pull request merged**